tf2onnx/tfonnx.py

```diff
@@ -21,8 +21,10 @@
 
 import tf2onnx
 from tf2onnx import utils
-from tf2onnx.function.select import select_op8
 from tf2onnx.function.gathernd import gathernd_op
+from tf2onnx.function.range import range_op7
+from tf2onnx.function.select import select_op8
+from tf2onnx.function.sparse_softmax_cross_entropy_with_logits import sparse_softmax_cross_entropy_with_logits_op
```

Comment:
this will be verbose if we move more complex ops to functions.

how about put xxxx_op that we want to export into ```tf2onnx/function/__init__.py```
and just ```from tf2onnx.function import *``` here